### ISSUE TITLE
[enhancement]: How do you launch it? 

### ISSUE THREAD

**Issue by DataJuggler** [invoke-ai/InvokeAI#7484]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What should this feature add?

I ran the installer and it ran once. Now I look in the folder and I see Invoke AI web.exe, and I tried to run that.

What file do I launch to run it? It used to be launch.bat.

### Alternatives

Just wondering why a programmer with 26 years experience cant figure out how to launch it.

### Additional Content

Why is there not a desktop shortcut created?

**Comment by DataJuggler** [comment]
I figured it out. You confused me by making me run an installer to launch it. Is there a way I can get a short cut to have a 1 step  
Launch?

Thanks

**Comment by jmk2142** [comment]
Late to the party but I had a related question. When launching from the launcher - I can't seem to get things like context support from my mouse when on the "app" for my Logitech MX Master.

_- Is there a particular file I can point to in Logi+ so I can program the mouse to run keyboard shortcuts in the Invoke UI?_

I'm also interested in whether I can get the Logitech Creative Console to be context aware of when I'm using Invoke's UI so I can put together a whole bunch of custom actions.

**Comment by psychedelicious** [comment]
@jmk2142 Good question - I would have guessed the launcher's executable is the right file. Does that not work?

PS: In the future, best to create a new GH issue or ask on [discord](https://discord.gg/ZmtBAhwWhy).

**Comment by jmk2142** [comment]
I haven't had much luck with it. I can see the Invoke - Community Edition Launcher in Logi Options+. Glorious fluorescent green icon and all. But assigning keys to it, doesn't activate it in the app. The global settings for the mouse continue to take effect. I'll fish around some more and then head over to the discord, put in a GH issue if I can't resolve it. Thanks!

**Comment by DataJuggler** [comment]
I had this same question when I first installed it. I thought the installer was just to install it. It also updates and launches it.
I created a folder called 'Launcher' and put the installer in it. I click the installer every time I launch it. 

<img width="367" height="141" alt="Image" src="https://github.com/user-attachments/assets/1cec54c3-a1a3-4650-8e16-d84afb7abe8c" />

When you click the installer, this windows pops up:

<img width="786" height="593" alt="Image" src="https://github.com/user-attachments/assets/0841bb8b-57ad-4ee6-bd13-1746bc240c60" />

Click Launch.

You can create a shortcut to the Installer by right clicking the installer, selecting 'Send To Desktop (Create Shortcut)

P.S. - I didn't know an update was available. I can't read the small font and didn't notice 'Version 6 is available' until I pasted it here'.

**Comment by psychedelicious** [comment]
@jmk2142 Please ping me (here or on discord) with the outcome. I have a logitech mouse in a box somewhere that uses Logi Options+ and can grab if you don't figure it out.

**Comment by jmk2142** [comment]
@psychedelicious Hey, sorry — been a busy few days. I’ll try to formalize my findings once I’ve done more testing, but here’s what I’ve discovered so far:

I haven’t been able to get the Logitech mouse (Options+) to contextually detect the WebUI launched via the default launcher.

Separately, I’ve been running into an issue where, after processing long queues through the Queue Agent, I receive a SIGTERM that terminates the server while the launcher remains active. It typically handles ~150 images fine, but then abruptly fails. When verbose debug logging and memory tracking are enabled, the issue happens even sooner — often around 50 images or fewer.

I’ve ruled out OOM, VRAM, and system RAM limitations — running on a 5090 with 64GB of RAM, there's still plenty of headroom. Adjusting TdrDelay (or disabling it entirely) seems to mitigate the issue slightly but doesn’t fully resolve it. From what I can tell based on process monitoring, the SIGTERM appears to originate from the launcher itself, not an external process.

As a workaround, I’ve been launching Invoke directly (invokeai-web --root PATH) and using Nativefier to wrap the GUI as its own app. This seems to allow Logitech Options+ to recognize it contextually, which gives me hope that the Creative Console MX will also be programmable. I might keep exploring Nativefier to inject custom JavaScript and create hotkey-friendly control layers, allowing tighter integration with external input devices.

I know that’s a lot of info from a simple Logitech issue, but I figured it was worth sharing — especially since you’ve been kind enough to reply. Probably deserves its own thread eventually. ;-)

**Comment by psychedelicious** [comment]
We believe the SIGTERM issue to be related to heap size limits imposed by Electron. Here's a long electron issue on the topic: https://github.com/electron/electron/issues/31330

TL;DR: Certain types of allocations have a max memory usage of 4GB, and when we go over that, we get a SIGTERM and the Chromium process (i.e. the UI) terminates.

I'm not sure how to resolve this, but as you discovered, running the UI in a browser window avoids the issue. You can also enable Server Mode in the launcher settings, which will run the python server via Launcher but let you access the UI w/ your browser.

I opened LogiOptions+ and can't figure out how to point it at the app either. Gotta be a way, because it works for other electron apps, maybe some electron config? Not having any luck looking this up 🤔
